### PR TITLE
fsplit

### PR DESCRIPTION
**Pull request by benjub**
As commit messages say.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -74348,13 +74347,34 @@ composition with a (2-place) operation to build compound operations such
...
+      anbi1i id wrel relcnv dfrel4v mpbi mptv 3eqtr4i ) AEZBEZFGUAZUBZHZABIZWBW
+      AWAJZKZABIWDALWGUJWEWHABWEWBWAWCHWBGMZWBWAFHZNZWHWAWBWCARZBRZUCGWBWAFWLUD
+      WKCEZWAKZWBWNWNJZKZNZCSZWHWBFUEZWAKZWQNZCSXAWQCSZNWSWKXAWQCUFXBWRCWQXAWOW
+      QWTWNWAWNWNWBCRZXDUGUHUIOXAWJXCWIFLUKZWBLMXAWJULLLFUMXEUNLLFUOUPWMLWBWAFU
+      QURWIWBWNDEZKZCDIZMWBWNXFJZKZXGNZDSZCSXCGXHWBCDUSUTXGCDWBVAXLWQCXLXFWNKZW
+      QNZDSWQXKXNDXKXGXJNXMXJNXNXJXGVCXGXMXJCDVBVMXMXJWQXMXIWPWBXFWNWNVDPVEQOWQ
+      WQDCXMWQVFTVGOVHVIVJWQWHCAWOWPWGWBWOWNWAWNWAWOVNZXOVKPTVGQVLWDVOWDWFKWCVP
+      ABWDVQVRABWGVSVT $.
+  $}
+
```
Comment (avekens) [inline set.mm:74371]: Neither ~fpar nor ~fsplit are currently used in proofs. So what are they good for? The given examples are also not very enlightening.

File: scripts/min.cmd [hunk 2]
```diff
@@ -37,7 +37,7 @@ match 1.tmp 'MM>' n
 match 1.tmp '  ' y
 break 1.tmp ''
 unduplicate 1.tmp
-add 1.tmp 'prove ' '$min xxx/allow */no_new ax-*$save new_proof/compressed$_exit_pa$'
+add 1.tmp 'prove ' '$min xxx/allow */no_new ax-*/may$min *$save new_proof/compressed$_exit_pa$'
```
Comment (avekens) [inline scripts/min.cmd:40]: What does this enhancement mean?

**Review by icecream17 — APPROVED** [review]
fpar and fsplit provide equivalences between two classes: one in "non-setvar notation" and the other in mpt/mpo notation. The only use I can see for this is to show alternate definitions in the sense of what [df-in](https://us.metamath.org/mpeuni/df-in.html) says about dfin2 and dfin4.

Regardless of usefulness the changes are improvements and bj-bixor is mathbox.

The example in fpar says,
F = sqrt & G = abs => ``(x (+ o. H) y) = (sqrt`x) + (abs`y)``

The example in fsplit says, (in combination with fpar)
F = sqrt & G = abs => ``(((+ o. H) o. ◡(1st ↾ I )) ` x) = (sqrt`x) + (abs`x)``

File: scripts/min.cmd (hunk 2 above)
Comment (benjub) [reply scripts/min.cmd:40]: Sorry, this was meant to be temporary and not added to the PR. I'm going to remove it.

**Comment by tirix** [comment]
As a remark, it might also be possible to get the same result using Mario's function operation map `oF`:
``( sqrt oF + abs ) = ( x e. CC |-> ( ( sqrt ` x ) + ( abs ` x ) ) )``

File: set.mm (hunk 1 above)
Comment (benjub) [reply set.mm:74371]: I haven't really considered applications. When looking at dfid2, I saw it had one discouraged use, so I just set on to remove it.

File: set.mm [hunk 3]
```diff
@@ -65983,9 +65981,9 @@ complete lattice has a (least) fixpoint.  Here we specialize this
      universe onto its power class.  Compare ~ canth that works for sets.
 
      This failure comes from a limitation of the collection principle (which is
-     necessary to avoid Russell's paradox ~ ru ): ` P _V ` , being a class,
-     cannot contain proper classes, so ` ~P _V ` , so it is no larger than
-     ` _V ` , which is why ` _I ` "succeeds" in being surjective onto ` ~P _V `
+     necessary to avoid Russell's paradox ~ ru ): ` ~ P _V ` , being a class,
```
Comment (tirix) [inline set.mm:65984]: Why split the `~` from the `P` here?
I suppose it should be `` ` ~P _V ` ``.

**Comment by benjub** [comment]
> As a remark, it might also be possible to get the same result using Mario's function operation map `oF`: `` ( sqrt oF + abs ) = ( x e. CC |-> ( ( sqrt ` x ) + ( abs ` x ) ) ) ``

Actually, it's rather https://us.metamath.org/mpeuni/df-of.html (the pointwise operation) which could be defined in terms of the diagonal embedding $x \mapsto (x, x)$, that is, if I call that embedding $\delta$, then
$(x \mapsto f(x) R g(x) ) = ( f (o_f R) g ) \circ \delta$.

File: set.mm (hunk 3 above)
Comment (benjub) [reply set.mm:65984]: The main reason to do that is because it's 1:30am !

Comment (benjub) [reply set.mm:65984]: Fixed.

File: scripts/min.cmd (hunk 2 above)
Comment (benjub) [reply scripts/min.cmd]: Done.

**Review by tirix — APPROVED** [review]

**Pull request merged**